components/bt/common/btc/core/btc_task.c

```diff
@@ -94,11 +94,11 @@ static const btc_func_t profile_tab[BTC_PID_NUM] = {
 #endif  ///BLE_INCLUDED == TRUE
     [BTC_PID_BLE_HID]     = {NULL, NULL},
     [BTC_PID_SPPLIKE]     = {NULL, NULL},
+    [BTC_PID_DM_SEC]      = {NULL,                        btc_dm_sec_cb_handler   },
```

Comment:
@PSONALl  If no functionality is changed in this file, please clean the diff and revert it

Reply:
@Sushant-Espressif, BLUFI profile ID (BTC_PID_BLUFI) is removed from bluedroid specific part i. e, from under the option: #ifdef CONFIG_BT_BLUEDROID_ENABLED, but this profile ID (BTC_PID_DM_SEC) was below it, so it was just removed from there and added at one line above, as it is bluedroid specific so it was kept under this option, thats why eventually this change is ocurring